device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubListener.java

```diff
@@ -11,7 +11,7 @@
 
 /**
  * Callback interface for communicating connection and message status updates from individual protocol clients
- * (mqtt, https, amqps) to the Tranpsort layer that handles queueing of messages and connecting/reconnecting/disconnecting
+ * (mqtt, https, amqps) to the Transport layer that handles queueing of messages and connecting/reconnecting/disconnecting
```

Comment:
As long as we're fixing this, I don't see why it would be capitalized. It isn't a proper noun, right?